### ISSUE TITLE
Color tree panel based on visited status

### ISSUE THREAD

**Issue by cons01e** [n0remac/Acute-Alligators-2021-Summer-Code-Jam#63]

**Comment by n0remac** [comment]
@cons01e Hello. I lost your contact information, are you still on discord? I lost my old account.

**Comment by cons01e** [comment]
@n0remac yeah I'm still on discord but my username has probably changed since then. you can reach me at menachemharrison@gmail.com if you want exchange contact info.
